File: C#/Basic/fibonacci.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter the number of Fibonacci terms: ");
        int n = int.Parse(Console.ReadLine());
        int a = 0, b = 1, c;

        Console.Write("Fibonacci Series: ");
        for (int i = 0; i < n; i++)
        {
            Console.Write(a + " ");
            c = a + b;
            a = b;
            b = c;
        }
        Console.WriteLine();
    }
}

File: C++/Basic/fibonacci.cpp
#include <iostream>

int main() {
    int n;
    std::cout << "Enter the number of Fibonacci terms: ";
    std::cin >> n;
    int a = 0, b = 1, c;

    std::cout << "Fibonacci Series: ";
    for (int i = 0; i < n; ++i) {
        std::cout << a << " ";
        c = a + b;
        a = b;
        b = c;
    }
    std::cout << std::endl;
    return 0;
}
